src/ddmd/globals.d

```diff
@@ -74,6 +74,7 @@ struct Param
 {
     bool obj;               // write object file
     bool link;              // perform link
+    bool compileimports;    // compile imported modules that have not been passed in
```

Comment:
The D modules and the C++ headers need to be kept in sync. You need to update `src/ddmd/globals.h` too.

Reply:
done, thanks for catching that.